examples/ios_app/CoreUtilsObjC/Answers.mm

```diff
@@ -1,10 +1,11 @@
 #import <CoreUtils/Answers.h>
+#include <limits>
 
 @implementation Answers
 
 - (NSInteger)answer {
     NSLog(@"%@", [Bar new].baz);
-    return 42;
+    return std::numeric_limits<int>::max();
```

Comment:
I'm not sure if there's a better case for exercising this. Even before setting `sdk_dylibs`, the previous code compiled successfully (after changing it to a `.mm`), so I imagine there's some autolinking going on for a `.mm` file. I only noticed that we were missing these flags when comparing two projects (one building with standard Xcode for many years and one with rules_xcodeproj).